.github/workflows/rancher-components.yaml

```diff
@@ -0,0 +1,25 @@
+name: rancher-components
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Please create an action by temporary removing the `master` flag, so we are sure to do not break `master` afterwards.
You can then link the [action](https://github.com/torchiaf/dashboard/actions/workflows/test.yaml) in the PR.

Reply:
Sorry I did not realize that is a PR and not a push, so it's already here 🙃 

For the records here's the checks:
https://github.com/rancher/dashboard/actions/workflows/rancher-components.yaml

Noticed also the check below 🙃